### DISCUSSION TITLE
antismash-db 3 ClusterCompare db

### DISCUSSION THREAD

**Discussion by nicholascdove** [antismash/antismash#654] [General]
Does an antismash-db 3 ClusterCompare database exist somewhere? If not, where can I download the database to make one?

**Comment by SJShaw** [comment]
The full set of the 3.0 results is available here: https://dl.secondarymetabolites.org/database/3.0/asdbv3_all_outputs.tar.bz2

**Comment by nicholascdove** [comment]
Great thank you! I'll share a link to the database when I am finished.
